aiida_firecrest/scheduler.py

```diff
@@ -290,34 +307,55 @@ def get_jobs(
             # therefore it requires some parsing, that is unnecessary now.
             # I just store is as a raw string for the moment, and I leave
             # this_job.allocated_machines undefined
-            # if this_job.job_state == JobState.RUNNING:
-            #     this_job.allocated_machines_raw = thisjob_dict['allocated_machines']
+            if this_job.job_state == JobState.RUNNING:
+                this_job.allocated_machines_raw = raw_result["nodelist"]
 
             this_job.queue_name = raw_result["partition"]
 
-            # try:
-            #     walltime = (self._convert_time(thisjob_dict['time_limit']))
-            #     this_job.requested_wallclock_time_seconds = walltime  # pylint: disable=invalid-name
-            # except ValueError:
-            #     self.logger.warning(f'Error parsing the time limit for job id {this_job.job_id}')
+            try:
+                time_left = self._convert_time(raw_result["time_left"])
+                start_time = self._convert_time(raw_result["start_time"])
+
+                if time_left is None or start_time is None:
+                    this_job.requested_wallclock_time_seconds = 0
+                else:
+                    this_job.requested_wallclock_time_seconds = time_left + start_time
+
+            except ValueError:
+                self.logger.warning(
+                    f"Couldn't parse the time limit for job id {this_job.job_id}"
+                )
 
             # Only if it is RUNNING; otherwise it is not meaningful,
             # and may be not set (in my test, it is set to zero)
-            # if this_job.job_state == JobState.RUNNING:
-            #     try:
-            #         this_job.wallclock_time_seconds = (self._convert_time(thisjob_dict['time_used']))
-            #     except ValueError:
-            #         self.logger.warning(f'Error parsing time_used for job id {this_job.job_id}')
+            if this_job.job_state == JobState.RUNNING:
+                try:
+                    wallclock_time_seconds = self._convert_time(
+                        raw_result["start_time"]
+                    )
+                    if wallclock_time_seconds is not None:
+                        this_job.wallclock_time_seconds = wallclock_time_seconds
+                    else:
+                        this_job.wallclock_time_seconds = 0
+                except ValueError:
+                    self.logger.warning(
+                        f"Couldn't parse time_used for job id {this_job.job_id}"
+                    )
 
+            # TODO: The block below is commented, because dispatch_time
```

Comment:
issues maybe with more explanation?

Reply:
sure, the same as above, how about this:
```suggestion
            # `aiida-ssh` transport plugin parses one additional key values: `dispatch_time`
            # from Slurm's `squeue` response. Which is the time that job has started running.     
            # `FirecREST` server doesn't return this time. 
            # however, I'm not aware of any crucial usecase in `aiida-core` for this `dispatch_time` 
            # Only if crucial please an issue on `FirecREST` repo for this
```

Reply:
I would make an issue out of this, because aiida need to somehow be aware that these are not set. The easiest way would be to set it to None, but I am not sure if some type checking complains, since the arguments are not set as optional. But in anycase this has to be handled somehow cleanly and since it needs a bit more time on the aiida-core side, an issue would be good.

Reply:
I think `aiida-core` sets values as `None` when cannot find it in the the output dict of this method. 
Therefore I just open an issue in `aiida-firecrest` just to have this documented there.